docs/sources/plugins/_index.md

```diff
@@ -4,5 +4,48 @@ type = "docs"
 [menu.docs]
 name = "Plugins"
 identifier = "plugins"
-weight = 8
+weight = 1
 +++
+
+# Plugins
+
+In addition to the wide range of visualizations and data sources that are available immediately after you install Grafana, you can extend your Grafana experience through _plugins_.
+
+You can [install]({{< relref "./installation.md" >}}) one of the plugins built by the Grafana community, or [build one yourself]({{< relref "./developing/development.md" >}}).
+
+Grafana supports three types of plugins: _panels_, _data sources_, and _apps_.
```

Comment:
It feels like we are missing a connection between this and the plugins page. Somewhere in here explain that you can filter plugins on the Grafana Plugins page by these types. What that page is missing is the concept information you have here.

Reply:
Ooh! We can link to the filtered views.

https://grafana.com/grafana/plugins?direction=asc&orderBy=weight&type=panel